package/uboot_custom/uboot_custom.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-UBOOT_CUSTOM_VERSION = 69b1326504844096af1988a1d6f1d3dd35185e70
+UBOOT_CUSTOM_VERSION = 19e204609c1ebc0f81e503d0775022d29b143346
```

Comment:
remember to upgrade against merged head (not super important but clean to keep it on head of swift-v2016.1